recipe/meta.yaml

```diff
@@ -10,21 +10,22 @@ source:
   md5: 9c2ad6cf86fe4a7bc0de8d2d92b101f4
 
 build:
-  number: 0
+  number: 1
   skip: True  # [win]
 
 requirements:
   build:
-    - fftw
-    - gcc
+    - {{ compiler('c') }}
+    - openmp  # [osx]
     - libtool
     - autoconf
     - automake
+  host:
+    - ncurses
```

Comment:
Why is `ncurses` needed?

Reply:
Dunno, I'll remove it and see what happens.

Reply:
The tests seem to link `-lncurses`, but a grep of the source tree didn't find it anywhere else:
https://github.com/NFFT/nfft/blob/7d24b6bb3ec9e185fddae4625403cdbf1f3fa7fa/tests/Makefile.am#L38
So far the CI seems to be passing without it explicitly included.